src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 2;
+  const SHORT_TERM_DISCOUNT = 20;
+  const PRICE_PER_DAY = 40;
+
+  let sum = days * PRICE_PER_DAY;
+
+  if (days < LONG_TERM && days > SHORT_TERM) {
+    sum = days * PRICE_PER_DAY - SHORT_TERM_DISCOUNT;
+  } else if (days >= LONG_TERM) {
+    sum = days * PRICE_PER_DAY - LONG_TERM_DISCOUNT;
+  }
```

Comment:
Prefer `if` with `return` over `if else` to simplify the code. Since you are returning `sum` at the end, you can return the discounted sum directly within the `if` blocks and avoid using `else if`.